tools/hangar/src/unsupported.test.ts

```diff
@@ -30,7 +30,7 @@ test("unsupported resource in target", async ({ expect }) => {
     },
   });
 
-  expect(sanitizeOutput(result.stderr)).toMatchInlineSnapshot('"runtime error: A Google Cloud region must be specified through the GOOGLE_REGION environment variable."');
+  expect(sanitizeOutput(result.stderr)).toMatchInlineSnapshot('"runtime error: Class extends value undefined is not a constructor or null"');
```

Comment:
Uh oh

Reply:
Ah yea this is probably because a test is not using the createApp helpers.

sooooo many snapshots to dig through 😂